source/nijigenerate/viewport/common/mesheditor/operations/base.d

```diff
@@ -65,13 +65,26 @@ public:
     bool mutateSelection = false;
     bool invertSelection = false;
     ulong maybeSelectOne;
+
+    // you should call updateVtxAtMouse() for updating vtxAtMouse
+    // because it also updates prevVtxAtMouse
+    // Note: prevVtxAtMouse for Undo may not safe to use currently
     ulong vtxAtMouse;
+    ulong prevVtxAtMouse;
```

Comment:
It seems `prevVtxAtMouse` is assigned, but not referred from anywhere now.
Should we use this?